<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab_bar_wnd</class>
 <widget class="QWidget" name="tab_bar_wnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>50</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(246, 247, 237);
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>6000</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel:hover{color:rgb(242, 190, 109);font-size:12px;}
QLabel{
font: 12pt &quot;微软雅黑&quot;;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="tab_label" name="labelMyHyperChain">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>MyChain</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="tab_label" name="labelChainBrowser">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>ChainBrowser</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="tab_label" name="labelChainAttestation">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>HC Register</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="tab_label" name="labelNodeSet">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>NodeSet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="tab_label" name="labelDevMode">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>DevMode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tab_label</class>
   <extends>QLabel</extends>
   <header>customui/tab_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
